### PR TITLE
Add support for Hive current_user function

### PR DESCRIPTION
**Pull request by losipiuk**

### CODE REVIEW

File: coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/HiveSqlConformance.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  public static SqlConformance HIVE_SQL = new HiveSqlConformance();
+
+  private HiveSqlConformance() {
+    super(PRAGMATIC_2003);
+  }
+
+  @Override
+  public boolean allowNiladicParentheses() {
+    return true;
+  }
```
Comment (wmoustafa) [inline coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/HiveSqlConformance.java:23]: Thanks for this change. @kxu1026 was about to publish a PR with a similar class (`HiveSqlConformance`). @kxu1026, please take a look and see if you would like to unify with this approach.

**Review by wmoustafa — COMMENTED** [review]
Thank you for the PR. Looks good overall. Let me sync with @kxu1026 about the SQL conformance and we should be good to go.

**Review by wmoustafa — APPROVED** [review]

**Comment by losipiuk** [comment]
Thanks @wmoustafa for quick merging! Are you planning on doing a release soon. From what I see the latest version uploaded to maven central is 1.0.12.

**Comment by losipiuk** [comment]
> Thanks @wmoustafa for quick merging! Are you planning doing a release soon? From what I see the latest version uploaded to maven central is 1.0.12.

@wmoustafa kind ping on this one :)
image

**Comment by wmoustafa** [comment]
@losipiuk Currently this is blocked on an issue being discussed here https://github.com/linkedin/coral/pull/36. We are trying to resolve that as soon as possible.

**Comment by losipiuk** [comment]
> @losipiuk Latest version is now published to Maven Central. https://repo1.maven.org/maven2/com/linkedin/coral/.

Thanks @wmoustafa . I will prepare PR to update dependency in Trino.

**Pull request merged**